### PR TITLE
Putting together the initial set up for the Scally Test Suite

### PR DESCRIPTION
**Pull request by chris-pearce**
@kllevin please review.

### CODE REVIEW

File: test/src/assets/scss/style.scss [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    @return $number;
+}
+@function rem($value) {
+    @if type-of($value) != "number" {
+        @error "You have to enter only a number for `$value`. You entered `#{$value}`.";
+    }
+    @return strip-unit($value) / strip-unit(16) * 1rem;
+}
+//@import '../../../../src/mixins-functions';
+
```
Comment (kllevin) [inline test/src/assets/scss/style.scss:60]: What's happening here?

File: test/src/assets/scss/vendor/_github-style-syntax-highlighting.scss [hunk 2]
```diff
@@ -0,0 +1,143 @@
+/**
```
Comment (kllevin) [inline test/src/assets/scss/vendor/_github-style-syntax-highlighting.scss:5]: Where is the normal sass heading?

File: test/src/data/base.json [hunk 3]
```diff
@@ -0,0 +1,217 @@
+{
+    "name": "Base",
+    "settings-location": "Base (Local)",
+    "docs-url": "#",
+    "source-code-url": "https://github.com/chris-pearce/scally/blob/master/src/_base.scss",
+    "tests": [{
+        "selector": "a",
+        "test": [{
+            "description": "Control <code class='b-code'>color</code>, <code class='b-code'>text-decoration</code> for both default and hover/focus states, and <code class='b-code'>transition</code>.",
+            "run": "<a href='http://campaignmonitor.com'>A link</a>",
```
Comment (kllevin) [inline test/src/data/base.json]: cm?

**Comment by kllevin** [comment]
LGTM

File: test/src/assets/scss/style.scss (hunk 1 above)
Comment (chris-pearce) [inline test/src/assets/scss/style.scss:60]: I haven't done the **Mixins & Functions** yet but the Test site is using the new `rem()` function so I added the functions here temporarily.

File: test/src/assets/scss/vendor/_github-style-syntax-highlighting.scss (hunk 2 above)
Comment (chris-pearce) [inline test/src/assets/scss/vendor/_github-style-syntax-highlighting.scss:5]: Will add it.

File: test/src/data/base.json (hunk 3 above)
Comment (chris-pearce) [inline test/src/data/base.json]: Changed.

**Pull request merged**